### ISSUE TITLE
Error in Inverse_Sigmoid_Loss ??

### ISSUE THREAD

**Issue by JoyMaz222** [zhangyp15/MonoFlex#41]
Hi, I was looking into the code and found some mismatch in Inverse_Sigmoid_Loss function. 
```
class Inverse_Sigmoid_Loss(nn.Module):
	def __init__(self):
		super(Inverse_Sigmoid_Loss, self).__init__()

	def forward(self, prediction, target, weight=None):
		trans_prediction = 1 / torch.sigmoid(target) - 1
		loss = F.l1_loss(trans_prediction, target, reduction='none')
		
		if weight is not None:
			loss = loss * weight

		return loss
```
This looks odd to me. `trans_prediction ` is calculated from `target` and L1 loss is calculated between `trans_prediction` and `target` which doesn't make sense. variable `prediction` wasn't even used in this function.  Any explanation for that ? Thanks
